docs/ecosystem/explorer.md

```diff
@@ -7,9 +7,131 @@ sidebar_position: 2
+1. View basic data such as transactions per minute, block height, total transactions, total accounts, application 
+quantity, side chain quantity, and more.
+2. Query specific information related to block height, transaction ID, and addresses.
+3. Contribute to the development of the aelf ecosystem by participating in activities like node elections, resource 
+purchases, and contract viewing.
 - Mainnet: https://explorer.aelf.io/
 - Testnet: https://explorer-test.aelf.io/
 
 ## 2. Using the Explorer
+
+### 1. View the overall status of the AElf blockchain.
+
+On the homepage, the first section displays essential data, including ELF current price, total transaction counts, 
+governance rewards, last block number, total account counts, and voter basic rewards. Below this section, real-time data 
+on the AElf ecosystem's operation can be found, such as the latest blocks, latest transactions, and transactions per 
+minute.
+
+![homepage](/img/explorer-homepage.png)
+
+### 2. Viewing information on the AElf chain
+
+In the top-left menu bar, there are four options. Under the second option 'blockchain,' there are three sub-options: 
+blocks, transactions, and address. With these three options, you can:
+- View information about all blocks on the AElf chain, including who mined the block and the transactions that occurred 
+in that block.
+- Explore each transaction that occurred on the AElf chain, including the sender and receiver addresses, as well as the 
+method called in the transaction.
+- See the top 1000 wallet addresses with the most ELF tokens on the AElf chain, along with all contracts deployed on the
+AElf chain.
+
+![blockchain](/img/explorer-blockchain.png)
+
+### 3. Viewing token symbols on AElf chain
+
+In the top-left menu bar, there are four options. Clicking on the third option 'token' will display all token symbols on
+the AElf chain, including symbols for both FT (Fungible Tokens) and NFT (Non-Fungible Tokens).
+
+![symbol](/img/explorer-symbol.png)
+
+### 4. Viewing Governance on AElf chain
+
+In the top-left menu bar, there are four options. Clicking on the fourth option 'governance' reveals three sub-options: 
+proposal, vote, and resource.
+
+#### Proposal
+
+Users can actively engage in on-chain governance by using the AElf blockchain explorer to view, initiate proposals, and 
+establish their own organizations. Initiating proposals and creating organizations require logging in using a browser 
+plugin.
+
+![proposal](/img/explorer-proposal.png)
+
+#### Vote
+
+Users can apply to become a candidate node and vote for nodes through the AElf blockchain explorer. 
+The scenario-based operations in the AElf blockchain explorer are as follows:
+
+(1) Applying to Become a Candidate Node
+
+Users want to become a production node on AElf, participate in block production, and maintain the orderly operation of 
+the entire AElf ecosystem. At this point, users need to apply to become a node on the AElf blockchain explorer.
```

Comment:
```suggestion
A node operator wants their node to become a production node on AElf, participate in block production, and maintain the orderly operation of the entire AElf ecosystem. At this point, the node owner needs to apply for their node to become a production node on the aelf blockchain. This can be done using the AElf blockchain explorer.
```